platforms/chibios/drivers/ws2812_pwm.c

```diff
@@ -40,15 +40,15 @@
 // Default Push Pull
 #ifndef WS2812_EXTERNAL_PULLUP
 #    if defined(USE_GPIOV1)
-#        define WS2812_OUTPUT_MODE PAL_MODE_STM32_ALTERNATE_PUSHPULL
+#        define WS2812_OUTPUT_MODE PAL_MODE_ALTERNATE_PUSHPULL
 #    else
-#        define WS2812_OUTPUT_MODE PAL_MODE_ALTERNATE(WS2812_PWM_PAL_MODE) | PAL_STM32_OTYPE_PUSHPULL | PAL_STM32_OSPEED_HIGHEST | PAL_STM32_PUPDR_FLOATING
+#        define WS2812_OUTPUT_MODE PAL_MODE_ALTERNATE(WS2812_PWM_PAL_MODE) | PAL_MODE_OUTPUT_PUSHPULL | PAL_OUTPUT_SPEED_HIGHEST | PAL_PUPDR_FLOATING
```

Comment:
Apparently this cannot work, because for GPIOv2
```c
#define PAL_MODE_ALTERNATE(n)           (PAL_STM32_MODE_ALTERNATE | PAL_STM32_ALTERNATE(n))
#define PAL_MODE_OUTPUT_PUSHPULL        (PAL_STM32_MODE_OUTPUT | PAL_STM32_OTYPE_PUSHPULL)
```
and `PAL_STM32_MODE_OUTPUT | PAL_STM32_MODE_ALTERNATE  == PAL_STM32_MODE_ANALOG`.

So this kind of usage would require some more generic defines like
```c
#define PAL_OUTPUT_TYPE_PUSHPULL  PAL_STM32_OTYPE_PUSHPULL
#define PAL_OUTPUT_TYPE_OPENDRAIN PAL_STM32_OTYPE_OPENDRAIN
```